auction-app-frontend/src/assets/style/home-page-product-tabs.scss

```diff
@@ -30,5 +30,23 @@
       border-bottom: 2px solid base.$primary-color;
       border-radius: 0;
     }
+
+    .loading-label {
+      color: #9B9B9B;
+      text-align: center;
+    }
+
+    .infinite-scroll-grid {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: center;
+      gap: 2%;
+
+      .grid-item {
+        flex-basis: 23.5%;
+        margin-bottom: 2%;
+      }
+    }
```

Comment:
Since we could have infinite scroll on some other pages (not only on home page), create `InfiniteScroll.scss` file and put this style definition there.